home/models.py

```diff
@@ -272,6 +272,11 @@ class WhatsappBlock(blocks.StructBlock):
         required=False,
         max_num=3,
     )
+    list_button_title = blocks.CharBlock(
```

Comment:
To avoid confusion with buttons, which are different from lists, perhaps `list_title` would be a better name?

(If we decide to make this change, we should do the same in the import/export code, etc.)

Reply:
Changed